### ISSUE TITLE
Run Tests is failing

### ISSUE THREAD

**Issue by jarq6c** [NOAA-OWP/hydrotools#152]
```console
=================================== FAILURES ===================================
_____________________ test_verify_client_session_signature _____________________

    def test_verify_client_session_signature():
        """Verify ClientSession signature was forged correctly. Note: does not check all
        valid params"""
        import forge
    
        sig = forge.fsignature(ClientSession)
>       assert {"retry", "n_retries", "cache", "headers"}.issubset(set(sig.parameters))
E       assert False
E        +  where False = <built-in method issubset of set object at 0x7ff8461b3f20>({'cache', 'kwargs', 'n_retries', 'retry'})
E        +    where <built-in method issubset of set object at 0x7ff8461b3f20> = {'cache', 'headers', 'n_retries', 'retry'}.issubset
E        +    and   {'cache', 'kwargs', 'n_retries', 'retry'} = set(mappingproxy(OrderedDict([('retry', <FParameter "retry:bool=True">), ('n_retries', <FParameter "n_retries:int=3">), ('cache', <FParameter "cache:CacheBackend=None">), ('kwargs', <FParameter "**kwargs">)])))
E        +      where mappingproxy(OrderedDict([('retry', <FParameter "retry:bool=True">), ('n_retries', <FParameter "n_retries:int=3">), ('cache', <FParameter "cache:CacheBackend=None">), ('kwargs', <FParameter "**kwargs">)])) = <FSignature (*, retry:bool=True, n_retries:int=3, cache:CacheBackend=None, **kwargs)>.parameters
```

**Comment by jarq6c** [comment]
@aaraney Not sure what's up yet. Recent merge of the updated `nwm_client_new` triggered a failure.

**Comment by jarq6c** [comment]
Appears to have been fixed by #154
